### ISSUE TITLE
Il est impératif de ne pas utiliser de système de gestion de version pour être conforme à la norme !

### ISSUE THREAD

**Issue by nmDejamobile** [la-rache/la-rache.com#9]
Enlever le site de github et envoyer une archive zip du site par email à qui demande les sources.
Les modifications pourront être traitées de la même façon.

**Comment by feraudet** [comment]
Il faudrait qu'il y ait des contributions déjà  -_-

**Comment by nmDejamobile** [comment]
Non, qu'il n'y ait pas de contributions car c'est trop compliqué de contribuer fait bien partie de la méthode. D'ailleurs, remonter des bugs également ne fait pas partie de la méthode... j'arrête tout de suite, promis.

**Comment by Youbze** [comment]
Ou éventuellement [GuyTeub](https://guyteub.com/), le seul agréé pour La RACHE.

**Comment by feraudet** [comment]
Excellente suggestion ! Nous avons étudié votre proposition avec attention lors d'une réunion de 3 heures sans ordre du jour précis.

Après mûre réflexion à La RACHE, nous avons décidé que :
- Migrer vers un système d'archives ZIP par email serait effectivement plus conforme à la norme
- Cependant, implémenter ce changement nécessiterait un effort qui va à l'encontre des principes de La RACHE
- De plus, GitHub étant devenu si populaire qu'il est maintenant mainstream, continuer à l'utiliser est paradoxalement une forme de résistance passive

Nous gardons donc Git, non par conviction, mais par inertie méthodologique. C'est très à La RACHE comme approche.

Merci pour votre contribution ! 🎯
